### ISSUE TITLE
Strange behaviour

### ISSUE THREAD

**Issue by takase1121** [felixse/FluentTerminal#471]
`clear` in PowerShell terminal does not entirely clear the terminal.  As an example: 
![image](https://user-images.githubusercontent.com/20792268/62408641-70c83380-b5fe-11e9-86b4-ae6c30e4016d.png)
You can see that I was just typing a commit message in `micro` and it does not clear correctly. What I need to do is press Enter repeatedly until the prompt overwrite the line I want to clear, then `clear` it. I am pretty sure that this is not the intended way to completely clear a console. Anyone had ideas to fix this?

**Comment by vasiby** [comment]
Yes, I also get occasional garbage at the bottom of the screen in WSL, and `clear` doesn't clean everything. It always happens on shell tab autocomplete when it lists possible launch parameters and they get permanently stuck until I move the prompt to the bottom. As a fix I  execute a full screen app such as `vim` and immediately exit it to clear the garbage

![image](https://user-images.githubusercontent.com/53339460/62779420-dcf1de00-babb-11e9-85d0-7bd1cdfbbcea.png)

**Comment by skipmeister123** [comment]
This drives me nuts.
